auditlog/views/auditlog_view.xml

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
add copyright in all XML files

Reply:
can you give me some example to add copyright in XML?
I never see MIS builder nor module_auto_update get this lines.

Reply:
It's a bit opinionated, some add, some don't. Usually it's the same as in code files, just inside `<!-- -->`. I'm pro-copyright in XML - it's still sort-of code and licensed thing